CONTRIBUTING.md

```diff
@@ -252,12 +252,7 @@ Our release process is as follows:
 2. Push this to a branch `release/vX.Y.Z` according to the version number of
    the anticipated release (e.g. `release/v0.18.0`) and open a **draft PR**.
 3. Bump all relevant versions in the codebase to the new version and push these
-   changes to the release PR. This includes:
-   1. `crates/ibc/Cargo.toml` file (making sure dependencies' versions are updated
-      too).
-   2. `crates/ibc/lib.rs` file documentation references' `html_root_url`
-      parameters must point to the new version.
-
+   changes to the release PR (`crates/ibc/Cargo.toml`, and other crates in `crates/`).
 4. In the `crates/ibc/` directory, run `cargo doc --all-features --open` locally to double-check that all the
```

Comment:
Just a friendly reminder. 
I think we can skip this step too since the `cargo doc` is already checked with the CI.